config/php.ini

```diff
@@ -0,0 +1,20 @@
+[PHP]
+date.timezone = UTC
+
+[CLI Server]
+cli_server.color = On;
+
+[Error Reporting]
+error_reporting = E_ALL | E_STRICT
+display_errors = Off
+log_errors = On
+track_errors = On
+display_startup_errors = On
+error_log = "./config/test.log"
+html_errors = On
+
+;[MySQL]
+; Trace mode. When trace_mode is active (=On), warnings for table/index scans and
+; SQL-Errors will be displayed.
+; http://php.net/mysql.trace-mode
+; mysql.trace_mode = Off
```

Comment:
I added that on. It was something I was testing for the previous story. The plugin just needed a pathfile for the .ini. So i created that directory. I looks like the Blogs site does something similar.  I just added some content that maybe would be useful down the line.